### PR TITLE
Feature/01 lobby

### PR DESCRIPTION
**Pull request by brauliodiez**
Basic lobby implementation (no UI design at all).

Work done:

- Added a fake api call to get the list of rooms.
- Created the basic elementso for the lobby page.
- Chopped down into subcomponents.
- Container encapsulation

### CODE REVIEW

File: 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+
+
+export const RoomListComponent = (props) =>
+  <List>
+    {props.rooms.map((room) =>
+      <ListItem button key={room.id}  onClick={(e) => props.onFieldChange('selectedRoom')(room)}>
```
Comment (brauliodiez) [inline 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx]: Should I place this event handling not inline?

File: 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          primary={room.name}
+        />
+      </ListItem>,
+    )}
+  </List>
+
+RoomListComponent.propTypes = {
+  rooms: PropTypes.array.isRequired,
+  onFieldChange : PropTypes.func.isRequired
+};
```
Comment (brauliodiez) [inline 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx]: Not sure if proptypes can be more detailed or concise, realized that there's a catch with Proptyes, you are adding extra wieght to your bundle (package plus all the js code, maybe on production mode is removed?) this doesn't happen on typescript ;)

File: 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx (hunk 1 above)
Comment (JaimeSalas) [reply 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx]: OMHO there is a lot of people talking about performance issues around this, but I got not idea if is bull sheet or really true. More than that do you really need curry on the function?

File: 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx (hunk 2 above)
Comment (JaimeSalas) [reply 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx]: Even you can make your own custom, but I don't know if fits the issue.

File: 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx (hunk 1 above)
Comment (JaimeSalas) [reply 00_ES6/00_start/src/pages/lobby/components/room-list.component.jsx]: Ok, never mind my prior comment.

**Pull request merged**